packages/dev/core/src/PostProcesses/motionBlurPostProcess.ts

```diff
@@ -135,8 +135,8 @@ export class MotionBlurPostProcess extends PostProcess {
         super(
             name,
             "motionBlur",
-            ["motionStrength", "motionScale", "screenSize", "inverseViewProjection", "prevViewProjection"],
-            ["velocitySampler", "textureSampler", "depthSampler"],
+            ["motionStrength", "motionScale", "screenSize", "inverseViewProjection", "prevViewProjection", "projection"],
+            ["velocitySampler", "depthSampler"],

```

Comment:
Note that I have removed `textureSampler` from the list as it is automatically added by the `PostProcess` constructor.